docs/misc/changelog.rst

```diff
@@ -45,6 +45,7 @@ New Features:
 - Added support for image observation when using ``HER``
 - Added ``replay_buffer_class`` and ``replay_buffer_kwargs`` arguments to off-policy algorithms
 - Added ``kl_divergence`` helper for ``Distribution`` classes (@09tangriro)
+- Added support for vector environments with num_envs > 1 (@benblack769)
```

Comment:
please add your username at the bottom too ;)

Reply:
Done